src/installer/corehost/cli/hostmisc/pal.h

```diff
@@ -46,6 +46,9 @@
 #define xout std::cout
 #define DIR_SEPARATOR '/'
 #define PATH_SEPARATOR ':'
+#ifdef _X
+#undef _X
+#endif
```

Comment:
```suggestion
#undef _X
```

`_X` is used all over the place, and I don't know having a longer name would help much.  Maybe `PAL_TEXT`, as that's the PAL version of the Windows `TEXT` macro?

Reply:
Sure, in the nearby context where this macro is used, the names are pretty self descriptive. If we are changing it to `PAL_TEXT`, should we rename it as part of this PR?

Reply:
Thinking a little bit more about this... something like this will touch every native code we have (especially if it's defined outside corehost), potentially requiring people to rebase and fix their outstanding PRs.  While I would like to see this happen (identifiers with names starting with underscores are reserved), if this change were to be made, it would have to be made in a separate PR with some discussion first.